### PR TITLE
Release Google.Cloud.Tools.SbomGenerator version 0.4.0

### PR DESCRIPTION
**Pull request by jskeet**
Changes since 0.3.0:

- Allows SBOMs to be generated from nupkg files without requiring a specific package namespace in the nuspec file

### CODE REVIEW

**Comment by jskeet** [comment]
(As ever, release one commit at a time - I thought I'd bundle the change and the release in one PR though.)

File: tools/Google.Cloud.Tools.SbomGenerator/NuspecPackage.cs [hunk 1]
```diff
@@ -45,4 +52,20 @@ public class NuspecMetadata
         [XmlElement("license")]
```
Comment (amanda-tarafa) [inline tools/Google.Cloud.Tools.SbomGenerator/NuspecPackage.cs:52]: nit: In a separate PR, maybe remove these as I believe they are not being used anymore?

**Review by amanda-tarafa — APPROVED** [review]

File: tools/Google.Cloud.Tools.SbomGenerator/NuspecPackage.cs (hunk 1 above)
Comment (jskeet) [reply tools/Google.Cloud.Tools.SbomGenerator/NuspecPackage.cs]: Done in this one.

**Review by amanda-tarafa — APPROVED** [review]

**Comment by amanda-tarafa** [comment]
I'll take a look at this.

**Comment by jskeet** [comment]
We re-released this in #12681

**Pull request merged**